Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp

```diff
@@ -148,6 +154,49 @@ DGeomHelpers::EmbedParameters *getETDG() {
   return new DGeomHelpers::EmbedParameters(DGeomHelpers::ETDG);
 }
 
+// void setCPCI(ForceFields::CrystalFF::CrystalFFDetails *self,
```

Comment:
please just remove this big block of commented-out code

Reply:
done